tests/main.sh

```diff
@@ -124,7 +124,7 @@ show_help() {
     exit 1
 }
 
-while getopts "hH?:vVtAsaxrlpS" opt; do
+while getopts "hH?vVtAs:a:x:rl:p:S:" opt; do
```

Comment:
You can just append a `:` after an option to have getopts put the flag arg into `$OPTARG`. This also allows getopts to handle all shifts for you.

Reply:
Agreed :+1: